contracts/DelFiPrice.sol

```diff
@@ -55,14 +55,14 @@ contract DelFiPrice is OpenOracleView {
     /// @notice The binary representation for 'REP' symbol, used for string comparison
     bytes32 constant symbolRep = keccak256(abi.encodePacked("REP"));
 
-    /// @notice The binary representation for 'BTC' symbol, used for string comparison
-    bytes32 constant symbolWbtc = keccak256(abi.encodePacked("BTC"));
+    /// @notice The highest ratio of the new source price to the anchor price that will still trigger the stored price to be updated
+    uint256 immutable upperBoundAnchorRatio;
 
-    /// @notice The binary representation for 'BAT' symbol, used for string comparison
-    bytes32 constant symbolBat = keccak256(abi.encodePacked("BAT"));
```

Comment:
we probably should still keep these constants for mapping.

Reply:
mm yes probably merged it wrong